### ISSUE TITLE
new version of swornguns

### ISSUE THREAD

**Issue by smajzy** [dmulloy2/SwornGuns#50]
i have test your last version of swornguns in 1.18.2 but the plugin didn't work, i don't know why, i have try many times. thank you

**Comment by smajzy** [comment]
if you have a solution or if you work it say me

**Comment by dmulloy2** [comment]
can you provide any more info? are there any stack traces in console?

**Comment by smajzy** [comment]
![image](https://user-images.githubusercontent.com/107326827/173227122-31ba8df5-48ce-417e-82aa-2a0934c3ae73.png)

**Comment by smajzy** [comment]
i have the fil .jat but not the case of the plugin

**Comment by smajzy** [comment]
and i have do the rebooot so i don't understand

**Comment by Log1x** [comment]
I think the problem they are having is `[19:28:33 WARN]: org.bukkit.plugin.InvalidPluginException: Cannot find main class `net.dmulloy2.swornguns.SwornGuns'`

It appears the shading is putting SwornGuns in `net.dmulloy2.swornguns.swornguns.SwornGuns` – the same thing is happening on UltimateArena.

**Comment by dmulloy2** [comment]
yeah i noticed that as well. weird thing is that it doesn't happen locally. not sure what's different with the jenkins env

**Comment by Log1x** [comment]
It happened when I built it as well.

As a temporary fix I changed [this](https://github.com/dmulloy2/SwornGuns/blob/master/pom.xml#L92-L94=) to:

```
<exclude>net.dmulloy2.swornguns.*</exclude>
<exclude>net.dmulloy2.swornrpg.*</exclude>
<exclude>net.dmulloy2.ultimatearena.*</exclude>
```

and it works but I don't know if it's technically correct.

**Comment by dmulloy2** [comment]
@Log1x interesting...i wonder what changed in the shade plugin

**Comment by Evan-Raponi** [comment]
Yup, also getting that error.

**Comment by DanFM1** [comment]
So how do you solve it anyway?
